AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm/README.md

```diff
@@ -1,18 +1,18 @@
-# `Genetic Algorithms on GPU using Intel® Distribution of Python numba-dpex` Sample
+# `Genetic Algorithms on GPU using Intel® Distribution for Python numba-dpex` Sample
 
-The `Genetic Algorithms on GPU using Intel® Distribution of Python numba-dpex` sample shows how to implement a general generic algorithm (GA) and offload computation to GPU using numba-dpex.
+The `Genetic Algorithms on GPU using Intel® Distribution for Python numba-dpex` sample shows how to implement a general genetic algorithm (GA) and offload computation to a GPU using numba-dpex.
 
-| Area                    | Description
+| Property                    | Description
 | :---                    | :---
+| Category                | Code Optimization
 | What you will learn     | How to implement the genetic algorithm using the Data-parallel Extension for Numba* (numba-dpex)?
 | Time to complete        | 8 minutes
-| Category                | Code Optimization
 
->**Note**: The libraries used in this sample are available in Intel® Distribution for Python* as part of the [Intel® AI Analytics Toolkit (AI Kit)](https://software.intel.com/en-us/oneapi/ai-kit).
+>**Note**: The libraries used in this sample are available in Intel® Distribution for Python* [Intel® Distribution for Python*](https://www.intel.com/content/www/us/en/developer/tools/oneapi/distribution-for-python.html).
```

Comment:
In the template we have a note about validation here. We can adjust it to IDP the following way:

> **Note**: This sample is validated on Intel® Distribution for Python* Offline Installer and AI Tools Offline Installer. For the full list of validated platforms refer to [Platform Validation](https://github.com/oneapi-src/oneAPI-samples/tree/master?tab=readme-ov-file#platform-validation).